packages/lesswrong/components/notifications/NotificationsMenuButton.tsx

```diff
@@ -196,69 +196,71 @@ const FriendlyNotificationsMenuButton = ({
   }, [toggle]);
 
   const {
-    LWTooltip, LWPopper, LWClickAwayListener, ForumIcon, NotificationsPopover,
+    LWPopper, LWClickAwayListener, ForumIcon, NotificationsPopover,
   } = Components;
   return (
     <div ref={anchorEl}>
-      <LWTooltip
-        title="Notifications"
-        placement="bottom"
-        popperClassName={classes.tooltip}
```

Comment:
I removed this tooltip because it was sometimes sticking open in a way that was annoying, and I think the icon will be familiar to people